examples/docs-wisdom-2/index.mjs

```diff
@@ -0,0 +1,32 @@
+import { loadStdlib } from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+
+if (process.argv.length < 3 || ['seller', 'buyer'].includes(process.argv[2]) == false) {
+  console.log('Usage: reach run index [seller|buyer]');
+  process.exit(0);
+}
```

Comment:
Something you need to think about and do with these programs is that the CI won't actual do any tests if you build it this way. You should do something like `rps-8-interact` does to script the two different versions to interact and run automatic tests. @DanBurton Please create a Jira card for this for Oliver to help Craig with on the CORE board.